test/stdlib/simd_diagnostics.swift

```diff
@@ -1,7 +1,7 @@
 // RUN: %target-typecheck-verify-swift
 
 // FIXME: No simd module on linux rdar://problem/20795411
-// XFAIL: OS=linux-gnu, OS=windows-msvc, OS=openbsd
+// XFAIL: OS=linux-gnu, OS=windows-msvc, OS=openbsd, OS=linux-android, OS=linux-androideabi
```

Comment:
Strangely, this test passes when the toolchain is built and run natively on Android but not when cross-compiling, not sure what's going on.

Reply:
And now it fails natively too. 🤷